SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialStrategySystem.cpp

```diff
@@ -94,24 +96,10 @@ void FSpatialStrategySystem::Advance(ISpatialOSWorker& Connection)
 	}
 
 	ActorSetSystem.Update(SetMemberView, RemovedEntities);
-	SetMemberView.ClearModified();
 
 	Strategy->Advance(Connection);
 }
 
-void FSpatialStrategySystem::ClearUserStorages()
-{
-	for (auto& Storage : UserDataStorages.DataStorages)
```

Comment:
Has this per entity custom data been fully removed, or reworked into something else?

Reply:
The user data storage and co are still around. It's just that now they clear their modification set at the beginning of their ::Advance function.